planex/cmd/createmockconfig.py

```diff
@@ -0,0 +1,123 @@
+"""
+planex-create-mock-config: Populate mock config with yum repositories
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import os.path
+import pprint
+import sys
+import yum
+
+import ConfigParser
+import StringIO
+
+import argcomplete
+
+from planex.util import add_common_parser_options
+from planex.util import setup_logging
+from planex.util import setup_sigint_handler
+
+
+def load_mock_reference(fname):
+    """
+    read in the reference mock configuration
```

Comment:
Could you please add a comment saying that this is the 'official' way to read the mock config, maybe with a reference to the mock code?